src/instance/search-config.yaml.example

```diff
@@ -41,3 +41,11 @@ indices:
     private: hm_antibodies
     elasticsearch:
       url: <ES URL>
+
+  hm_files:
+    active: true
+    reindex_enabled: false
+    public: hm_files
+    elasticsearch:
+      url: <ES URL>
+      mappings: "hubmap_translation/search-default-config.yaml"
```

Comment:
We probably need a file-api specific config yaml, @maxsibilla what do you think?

Reply:
@yuanzhou I don't have much experience with the antibodies ES instance so I don't know if the files component breaks that convention. It might be best to go forward with a separate config file just for the reason that it does not have a private and public index and the `main` function in `hubmap_translator.py` might have to be rewritten to natively support the `hm_files` section